Subdivision_method_3/include/CGAL/Subdivision_method_3/subdivision_methods_3.h

```diff
@@ -199,6 +199,57 @@ void Loop_subdivision(PolygonMesh& pmesh, const NamedParameters& np = parameters
 }
 // -----------------------------------------------------------------------------
 
+/*!
+ *
+ * applies an  subdivision several times on the control mesh `pmesh`.
+ * The geometry of the refined mesh is computed by the geometry policy mask `Linear_mask_3`,
+ * which is similar to Loop subdivision but does not change the shape of the mesh (only the connectivity).
+ * The new vertices are trivially computed as the average of the incident vertices (e.g., midpoint of an edge).
+ * This function overwrites the control mesh `pmesh` with the subdivided mesh.
+
+ * @tparam PolygonMesh a model of `MutableFaceGraph`
+ * @tparam NamedParameters a sequence of \ref bgl_namedparameters "Named Parameters"
+ *
+ * @param pmesh a polygon mesh
+ * @param np an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+ *
+ * \cgalNamedParamsBegin
+ *   \cgalParamNBegin{vertex_point_map}
+ *     \cgalParamDescription{a property map associating points to the vertices of `pmesh`}
+ *     \cgalParamType{a class model of `ReadWritePropertyMap` with `boost::graph_traits<PolygonMesh>::%vertex_descriptor`
+ *                    as key type and `%Point_3` as value type}
+ *     \cgalParamDefault{`boost::get(CGAL::vertex_point, pmesh)`}
+ *     \cgalParamExtra{If this parameter is omitted, an internal property map for `CGAL::vertex_point_t`
+ *                     should be available for the vertices of `pmesh`.}
```

Comment:
"should"  is a bad idea for something that is required.   It must be "must" -- and probably not only here.